cmd/influx/main.go

```diff
@@ -30,6 +30,8 @@ var (
 const (
 	// defaultFormat is the default format of the results when issuing queries
 	defaultFormat = "column"
+	
+	defaultPrecision = ""
```

Comment:
Actually, the default precision is actually nanos, right? So can you set this correctly?